src/services/contact.js

```diff
@@ -0,0 +1,29 @@
+import React, { useRef } from 'react';
+import emailjs from '@emailjs/browser';
+
+export const ContactUs = () => {
+    const form = useRef();
+
+    const sendEmail = (e) => {
+        e.preventDefault();
+
+        emailjs.sendForm('service_iwqsu3f', 'template_d0s4lc6', form.current, 'SMrzCAgEF5PcNX7Jo')
```

Comment:
It’s recommended to save Emailjs credentials in Environment variable file.

Reply:
Kutay said no need for that

Reply:
@MARWAHAKMI yeah, let's build it the way it works for now, after the project is finished we might move every credential in the env file, therefore we could get a more professional look. Bur for now Im not pushing anybody to do that.

Reply:
I pushed the changes if you can check them please

Reply:
@OmarBassatni97 you need to ask for review again in that case, github will notify me again.